Pipfile

```diff
@@ -11,6 +11,7 @@ docker = "*"
 Fabric = ">=1.13.1,<2.0"
 # https://github.com/docker/compose/issues/6839
 colorama = {version = ">=0.4, <1", markers = "sys_platform == 'win32'"}
+pypiwin32 = {version = "*", markers = "sys_platform == 'win32'"}
```

Comment:
@lyrixx we could add this to #61

Reply:
It was [already added](https://github.com/jolicode/docker-starter/pull/61/files#diff-1e61a31bf9b94805f869dc4137ec1885R12), thanks (and a bit simpler :joy:)